Google.Cloud.EntityFrameworkCore.Spanner/Extensions/TypeExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿#nullable enable
```

Comment:
nit, also here: Copyright header. Also, is the `nullable enable` necessary here?

Reply:
Thanks @olavloite , will add the copyright header. As far as the `#nullable enable`, no, it is not absolutely necessary. However, it allows for greater type safety at compile time, by letting us declare reference types as nullable (e.g., `MyType?`.)

Importantly, the feature can be enabled on a file-by-file basis by adding the `#nullable enable` directive. My thought was that I should take advantage of it with this new contribution, even if it wasn't being used everywhere. But I can certainly remove it if needed.

Microsoft has good documentation here: https://learn.microsoft.com/en-us/dotnet/csharp/nullable-references